### ISSUE TITLE
Typokana fails to run after upgrading to Xcode 11.4

### ISSUE THREAD

**Issue by nicholaslythall** [ezura/spell-checker-for-swift#16]
Running typokana produces the following output:
```
The operation couldn’t be completed. (SwiftSyntax.ParserError error 1.)
```

The problem was fairly easy to fix, just needed to update the version of [apple/swift-syntax](https://github.com/apple/swift-syntax) to`0.50200.0` both here and [ezura/SwiftSyntaxExtensions](https://github.com/ezura/SwiftSyntaxExtensions) and do a few minor code changes.

I'm putting the PRs for these changes shortly.

**Comment by Ahmad-Atef-Ali** [comment]
Thank you 👍 
waiting for your PR :)

**Comment by ezura** [comment]
Sorry for the inconvenience...!
This issue has been resolved from `5.2.0`.
Thanks to @nicholaslythall !
